projects/politie/ngx-sherlock/src/lib/value-for.pipe.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    ChangeDetectorRef,
+    OnDestroy,
+    Pipe,
+    PipeTransform,
+} from "@angular/core";
+import { Derivable, isDerivable, unresolved } from "@politie/sherlock";
+import { DerivableProxy, isDerivableProxy } from "@politie/sherlock-proxy";
+
+/**
+ * The {@link ValueForPipe} is designed to be used inside Angular's
+ * `@for` loops. It keeps a `Derivable` or `DerivableProxy` in sync with
+ * Angular's change detection without unwrapping the value directly.
+ *
+ * Unlike {@link ValuePipe}, this pipe **returns the derivable object itself**,
+ * so you can decide in the template how to extract the value (`.get()` or `$value`).
+ *
+ * Whenever the derivable updates, the `ChangeDetectorRef` of the `@Host()` component
+ * will be `markedForCheck` to ensure the loop items re-render correctly.
```

Comment:
The function is called `markForCheck`. Please correct to either `markForCheck` or marked for check (written out).